apartmenttherapy/order-service
# Elaine/orders count

#### What does this PR do & why?
Adding a query to get the total number of orders for the passed parameters.

#### How can this be manually tested?
- Tests should run.
- Deployed to dev for testing.
- In graphiql (https://order-service-dot-marketplace-dev-208414.appspot.com/graphiql) run the following (just an example):
```
query {
  ordersCount(customerUserId:"auth0|5a6a41518c931c652bb71f5e", orderStatus:[PENDING_ACCEPTANCE,ACCEPTED], passport:{traveler:"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuaWNrbmFtZSI6ImVsYWluZV9wcmltYXJ5IiwibmFtZSI6ImVsYWluZUBhcGFydG1lbnR0aGVyYXB5LmNvbSIsInBpY3R1cmUiOiJodHRwczovL3MuZ3JhdmF0YXIuY29tL2F2YXRhci9jYzhkNjJkMmE1NGI2NjM4ZjlhYzQyZWZmZWM1MjcwYz9zPTQ4MCZyPXBnJmQ9aHR0cHMlM0ElMkYlMkZjZG4uYXV0aDAuY29tJTJGYXZhdGFycyUyRmVsLnBuZyIsInVwZGF0ZWRfYXQiOiIyMDE5LTA0LTMwVDE2OjAwOjE2LjM2MVoiLCJlbWFpbCI6ImVsYWluZUBhcGFydG1lbnR0aGVyYXB5LmNvbSIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJpc3MiOiJodHRwczovL2Rldi1hdXRoLmFwYXJ0bWVudHRoZXJhcHltZWRpYS5jb20vIiwic3ViIjoiYXV0aDB8NWE2YTQxNTE4YzkzMWM2NTJiYjcxZjVlIiwiYXVkIjoiR2dLbjJoU3F5Zk5pNnRURlQxQmVkb1RFR1czbTdXcnciLCJpYXQiOjE1NTY2NDAwMTYsImV4cCI6MTU1OTIzMjAxNiwiYXRfaGFzaCI6IjNwRUpQRkN2QXYxVVYwWjF4ZnJKeWciLCJub25jZSI6ImZ1V2d3QWs1WHVLUiJ9.diCMwcRlehlKZXs0s1CYJDq-OT0ijGFfZibg9cAM3FU"})
}
```

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*


#### Link to story card and other documentation
https://app.asana.com/0/694906336753887/1122588044002343

#### GIF


#### Have you remembered to...?
